### PR TITLE
Initializes the github.com/BTrDB/btrdb/v5 module

### PR DESCRIPTION
**Pull request by alexmspina**

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -0,0 +1,16 @@
+module github.com/BTrDB/btrdb/v5
+
+go 1.14
+
+require (
+	github.com/BTrDB/btrdb v5.9.1+incompatible
```
Comment (immesys) [inline go.mod]: Yeh we should ensure this doesn't happen

File: go.mod [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+	github.com/golang/protobuf v1.4.0
+	github.com/grpc-ecosystem/grpc-gateway v1.14.4
```
Comment (immesys) [inline go.mod]: Where does this come from? That's odd. Maybe from the v5api proto file

File: go.mod [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+	github.com/huichen/murmur v0.0.0-20130808212358-e0489551cf51
+	github.com/op/go-logging v0.0.0-20160315200505-970db520ece7
+	github.com/pborman/uuid v1.2.0
+	golang.org/x/net v0.0.0-20200501053045-e0ff5e5a1de5
+	google.golang.org/genproto v0.0.0-20200430143042-b979b6f78d84
+	google.golang.org/grpc v1.29.1
```
Comment (immesys) [inline go.mod]: Can you change this to 1.23.1 ? There's a bug in 1.29.1

File: go.mod [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+	gopkg.in/BTrDB/btrdb.v5 v5.9.1 // indirect
```
Comment (immesys) [inline go.mod]: This should also not be there

**Review by immesys — CHANGES_REQUESTED** [review]
Looks close

File: go.mod (hunk 2 above)
Comment (immesys) [reply go.mod]: Yeah it's from the annotations. That's fine, you can leave that there

File: go.mod (hunk 1 above)
Comment (alexmspina) [reply go.mod]: removed after refactoring insert_test.go

File: go.mod (hunk 3 above)
Comment (alexmspina) [reply go.mod]: done

File: go.mod (hunk 4 above)
Comment (alexmspina) [reply go.mod]: removed after refactoring insert_test.go

**Review by immesys — APPROVED** [review]

**Pull request merged**